microhab/SPEXpdd.tex

```diff
@@ -0,0 +1,253 @@
+  % What path do you anticipate the project to take?
+Documentation for each step of the respected engineering paths is required. It
+is important during the implementation aspect to retain problems experienced and
+the solutions used to overcome.
+
+\subsection{Deliverables}
+\label{subsec:deliverables}
+  % When all is said and done, what will you have to show for it?
+  % Examples: Hardware, software, poster, ImagineRIT demo, presentations, technical papers...
+\begin{itemize}
+    \item Bill of Materials for Hardware
+    \item PCB schematics
+    \item Arduino software source files
+    \item C software source files
+    \item Fully assembled balloon, parachute, and payload
+\end{itemize}
+
+\subsection{Milestones}
+\label{subsec:milestones}
+  % Be as detailed as you can, but it's okay if there are unknowns.
+  % At the very least, specify how many semester you expect the project to take until it reaches completion.
+\subsubsection*{Electrical}
+\begin{itemize}
+    \item 2 weeks --- PCB design
+    \item 1 week --- PCB reviews
+    \item 1 week --- Order hardware
+    \item 1 week --- PCB assembly
+    \item 1 week --- PCB testing
+\end{itemize}
+
+\subsubsection*{Arduino software}
+\begin{itemize}
+    \item 1 week --- gather libraries needed
+    \item 1 week --- plan out software design and flow
+    \item 2 weeks --- write software
+    \item 1 week --- test software
+    \item 1 week --- neaten up software
+\end{itemize}
+
+\subsubsection*{C software}
+\begin{itemize}
+    \item 1 week --- prepare standard development environment
+    \item 1 week --- plan out software design and flow
+    \item 4 weeks --- create drivers
+    \item 8 weeks --- write software
+    \item 2 weeks --- test software
+    \item 1 week --- neaten up software
+\end{itemize}
+\section{Externalities}
+  % Things not directly related to the work or outcomes, but related to the project as a whole.
+\subsection{Prerequisite Skills}
+  % Which skills do team members need to have before work can start (not including skills that will be learned ``on the job'')?
+
+  % Estimate costs that would be needed to meet objectives.
+Covered in Accessibility.
+\subsection{Faculty Support}
+  % Identify faculty that will be involved (or would need to be involved) to meet objectives.
+  % Note that if a professor is the Principal Investigator (P.I.) for a project, there still needs to be a student as the SPEX Project Champion.
+\textmu{}Hab should be able to be completely engineered by students. Faculty
+support could be helpful for hardware testing. Also access to surface mounting
+labs would need to be gained through faculty as well.
+\subsection{Funding Requirements}
+As the purpose of the project is to minimize costs of launching a HAB to the
+fullest extent, funding requirements will be minimal. PCB print costs should be
+free due to a HAB sponsorship. Lighter balloons can cost less than 25USD, Helium
+price is hard to calculate but will also be significantly cheaper.
```

Comment:
Expand on this. This section is a core pillar to the whole idea of a microhab and needs some more thought or explanation.

You're proposing a low-cost, smaller, simpler alternative to a HAB. You have explained the smaller, simpler part, but what about cost? Convince me that microhabs are worth it instead of multiple experiments on a larger hab or on non-flight dev boards.